apps/locale/locale.html

```diff
@@ -212,7 +209,7 @@
     }
     return s.substr(0, i + 3) + r + (d ? '${locale.decimal_point}' + d: '');
   },
-  currency: n => ${currency},
+  currency: n => {console.log("Warning: Currency information is deprecated");return ${js("£")}+exports.number(n)},
```

Comment:
What do you think of:

```suggestion
  currency: n => {console.warn("Warning: Currency information is deprecated");return ${js("£")}+exports.number(n)},
```

Reply:
I agree, but doing that would make the package incompatible with most versions of Espruino,  `console.warn` was added this year: https://github.com/espruino/Espruino/issues/1302#issuecomment-1873917322

Given that `console.warn` is just an alias of `console.log`, I don't think it is worth it.

Reply:
Yeah fair point 👍